pyvista/plotting/render_window_interactor.py

```diff
@@ -1111,7 +1111,7 @@ def _simulate_keypress(self, key):  # pragma:
         """Simulate a keypress."""
         if len(key) > 1:
             raise ValueError('Only accepts a single key')
-        self.interactor.SetKeyCode(key)
+        self.interactor.SetKeySym(key)
```

Comment:
you may want to set both imo.

Reply:
Oh, thanks @mwestphal!

```suggestion
        self.interactor.SetKeyCode(key)
        self.interactor.SetKeySym(key)
```